src/vanilla/utils/atomFamily.ts

```diff
@@ -43,6 +45,7 @@ export function atomFamily<Param, AtomType extends Atom<unknown>>(
     }
 
     const newAtom = initializeAtom(param)
+    ;(newAtom as any)[SymbolOriginator] = createAtom
```

Comment:
Is `createAtom` just used as a reference key?
It doesn't seem very straightforward and useful.